src/Broadway/EventStore/EventStoreInterface.php

```diff
@@ -28,6 +28,8 @@ public function load($id);
     /**
      * @param mixed                      $id
      * @param DomainEventStreamInterface $eventStream
+     *
+     * @throws EventStoreException
```

Comment:
Makes sense to me. Since changes are so minor I'll await feedback from qandidate before fixing it.

Also thanks for referencing to "(un)checked" exceptions, didn't know this concept had a name :)